msal/src/main/java/com/microsoft/identity/client/PublicClientApplicationConfiguration.java

```diff
@@ -367,6 +373,10 @@ public void setPowerOptCheckEnabled(Boolean powerOptCheckEnabled) {
         this.powerOptCheckEnabled = powerOptCheckEnabled;
     }
 
+    public Boolean authorizationInCurrentTask() {
```

Comment:
Following the pattern for other booleans...

```suggestion
    public Boolean isAuthorizationInCurrentTask() {
```